src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-snapshot-list/rbd-snapshot-actions.model.ts

```diff
@@ -5,7 +5,8 @@ export class RbdSnapshotActionsModel {
   create: CdTableAction = {
     permission: 'create',
     icon: 'fa-plus',
-    buttonCondition: (selection: CdTableSelection) => !selection.hasSingleSelection,
+    buttonCondition: (selection: CdTableSelection) =>
```

Comment:
This can be removed as it's the default behavior, because multi selections are not allowed.

Reply:
Won't removing this keep "Create" always as the default button action?